.github/workflows/main.yml

```diff
@@ -57,6 +57,8 @@ jobs:
       run : | 
         sudo apt-get update
         sudo apt-get install -y libvirt-dev
+    - name: download deps
+      run : go mod download
```

Comment:
not related to this issue, but now that we are at it lets add
this environment variable for faster downloads GOPROXY=https://proxy.golang.org

as we do in https://github.com/kubernetes/minikube/blob/master/.travis.yml#L9